package.json

```diff
@@ -127,6 +127,7 @@
     "eslint-plugin-react-native": "3.5.0",
     "eslint-plugin-relay": "0.0.28",
     "flow-bin": "^0.94.0",
+    "flow-remove-types": "^1.2.3",
```

Comment:
what is this for? is it to run the codegen?

Reply:
Yes, for codegen.